src/transformers/models/layoutlmv3/configuration_layoutlmv3.py

```diff
@@ -14,7 +14,17 @@
 # limitations under the License.
 """ LayoutLMv3 model configuration"""
 
+from collections import OrderedDict
+from typing import Any, Mapping, Optional
+
+from packaging import version
+
+from transformers import TensorType
```

Comment:
No absolute import here as this breaks everything. This should be imported from `...utils`. Also, it's only used for type checking, so use forward references in the type annotations and import this in an `if TYPE_CHECKING:` block.